### ISSUE TITLE
Testbed deployment failing to setup ceph

### ISSUE THREAD

**Issue by osfrickler** [osism/issues#259]
Since yesterday, new testbed deployments are failing during the ceph installation:

```
2022-07-12 04:01:18.785863 | orchestrator | TASK [ceph-config : set osd_memory_target] *************************************
2022-07-12 04:01:18.785871 | orchestrator | fatal: [testbed-node-0.testbed.osism.xyz -> testbed-node-0.testbed.osism.xyz]: FAILED! => {"changed": false, "cmd": ["docker", "run", "--rm", "--net=host", "-v", "/etc/ceph:/etc/ceph:z", "-v", "/var/lib/ceph:/var/lib/ceph:z", "-v", "/var/run/ceph:/var/run/ceph:z", "--entrypoint=ceph", "quay.io/osism/ceph-daemon:pacific", "config", "set", "osd/host:testbed-node-0.testbed.osism.xyz", "osd_memory_target", "5892577689"], "delta": "0:00:00.885604", "end": "2022-07-12 04:01:17.447142", "msg": "non-zero return code", "rc": 1, "start": "2022-07-12 04:01:16.561538", "stderr": "Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)", "stderr_lines": ["Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)"], "stdout": "", "stdout_lines": []}
2022-07-12 04:01:18.785890 | orchestrator | fatal: [testbed-node-1.testbed.osism.xyz -> testbed-node-0.testbed.osism.xyz]: FAILED! => {"changed": false, "cmd": ["docker", "run", "--rm", "--net=host", "-v", "/etc/ceph:/etc/ceph:z", "-v", "/var/lib/ceph:/var/lib/ceph:z", "-v", "/var/run/ceph:/var/run/ceph:z", "--entrypoint=ceph", "quay.io/osism/ceph-daemon:pacific", "config", "set", "osd/host:testbed-node-1.testbed.osism.xyz", "osd_memory_target", "5892577689"], "delta": "0:00:01.027434", "end": "2022-07-12 04:01:17.809220", "msg": "non-zero return code", "rc": 1, "start": "2022-07-12 04:01:16.781786", "stderr": "Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)", "stderr_lines": ["Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)"], "stdout": "", "stdout_lines": []}
2022-07-12 04:01:18.785899 | orchestrator | fatal: [testbed-node-2.testbed.osism.xyz -> testbed-node-0.testbed.osism.xyz]: FAILED! => {"changed": false, "cmd": ["docker", "run", "--rm", "--net=host", "-v", "/etc/ceph:/etc/ceph:z", "-v", "/var/lib/ceph:/var/lib/ceph:z", "-v", "/var/run/ceph:/var/run/ceph:z", "--entrypoint=ceph", "quay.io/osism/ceph-daemon:pacific", "config", "set", "osd/host:testbed-node-2.testbed.osism.xyz", "osd_memory_target", "5892577689"], "delta": "0:00:00.922227", "end": "2022-07-12 04:01:17.849761", "msg": "non-zero return code", "rc": 1, "start": "2022-07-12 04:01:16.927534", "stderr": "Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)", "stderr_lines": ["Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)"], "stdout": "", "stdout_lines": []}
```
see e.g. https://zuul.osism.xyz/t/osism/build/23bae3b3d8814f9dbb69bf6c5f972fb8

**Comment by osfrickler** [comment]
Only pacific seems to be affected, deploying with octopus does not exhibit this failure. The issue seems related to `/etc/ceph` not getting populated on the target nodes.

**Comment by osfrickler** [comment]
Pacific (== stable-6.0) is still waiting for this fix to be merged https://github.com/ceph/ceph-ansible/pull/7255 , the corresponding fix is already implemented in stable-5.0.

**Comment by osfrickler** [comment]
Fix is merged, container rebuilt, working deployment verified.
